website/docs/usage/v3-1.md

```diff
@@ -0,0 +1,186 @@
+
+[![Prodigy: example of the new manual spans UI](../images/prodigy_spans-manual.jpg)](https://support.prodi.gy/t/3861)
+
+The upcoming version of our annotation tool [Prodigy](https://prodi.gy)
+(currently available as a [pre-release](https://support.prodi.gy/t/3861) for all
+users) features a [new workflow and UI](https://support.prodi.gy/t/3861) for
+annotating overlapping and nested spans. You can use it to create training data
+for spaCy's `SpanCategorizer` component.
+
+</Infobox>
+
+### Update the entity recognizer with partial incorrect annotations {#negative-samples}
+
+> #### config.cfg (excerpt)
+>
+> ```ini
+> [components.ner]
+> factory = "ner"
+> incorrect_spans_key = "incorrect_spans"
+> moves = null
+> update_with_oracle_cut_size = 100
+> ```
+
+The [`EntityRecognizer`](/api/entityrecognizer) can now be updated with known
+incorrect annotations, which lets you take advantage of partial and sparse data.
+For example, you'll be able to use the information that certain spans of text
+are definitely **not** `PERSON` entities, without having to provide the complete
+gold-standard annotations for the given example. The incorrect span annotations
+can be added via the [`Doc.spans`](/api/doc#spans) in the training data under
+the key defined as [`incorrect_spans_key`](/api/entityrecognizer#init) in the
+component config.
+
+```python
+train_doc = nlp.make_doc("Barack Obama was born in Hawaii.")
+# The doc.spans key can be defined in the config
+train_doc.spans["incorrect_spans"] = [
+  Span(doc, 0, 2, label="ORG"),
+  Span(doc, 5, 6, label="PRODUCT")
+]
+```
+
+<!-- TODO: more details and/or example project? -->
+
+### New pipeline packages for Catalan and Danish {#pipeline-packages}
+
+spaCy v3.1 adds 5 new pipeline packages, including a new core family for Catalan
+and a new transformer-based pipeline for Danish using the
+[`danish-bert-botxo`](http://huggingface.co/Maltehb/danish-bert-botxo) weights.
+See the [models directory](/models) for an overview of all available trained
+pipelines and the [training guide](/usage/training) for details on how to train
+your own.
+
+<!-- TODO: thank contributors and update with final numbers -->
+
+| Package                                           | Language | Tagger | Parser |  NER |
+| ------------------------------------------------- | -------- | -----: | -----: | ---: |
+| [`ca_core_news_sm`](/models/ca#ca_core_news_sm)   | Catalan  |        |        |      |
+| [`ca_core_news_md`](/models/ca#ca_core_news_md)   | Catalan  |        |        |      |
+| [`ca_core_news_lg`](/models/ca#ca_core_news_lg)   | Catalan  |        |        |      |
+| [`ca_core_news_trf`](/models/ca#ca_core_news_trf) | Catalan  |        |        |      |
+| [`da_core_news_trf`](/models/da#da_core_news_trf) | Danish   |        |        |      |
+
+### Resizable text classification architectures {#resizable-textcat}
+
+Previously, a trained [`TextCategorizer`](/api/textcategorizer) architectures
```

Comment:
```suggestion
Previously, a trained [`TextCategorizer`](/api/textcategorizer) architecture
```

Reply:
Maybe this should actually be "model"? A "trained architecture" is kinda weird. Actually, the whole sentence is weird 😂

How about:

```
Previously, the [`TextCategorizer`](/api/textcategorizer) architectures could not be resized, meaning that you couldn't add new labels to an already trained model.
```

Reply:
Sounds good!